client/lib/signup/step-actions/index.js

```diff
@@ -437,6 +437,35 @@ export function setOptionsOnSite( callback, { siteSlug, siteTitle, tagline } ) {
 	);
 }
 
+export function setStoreFeatures( callback, { siteSlug } ) {
+	if ( ! siteSlug ) {
+		defer( callback );
+		return;
+	}
+
+	wpcom.req
+		.post( `/sites/${ siteSlug }/themes/mine`, { theme: 'zoologist', dont_change_homepage: true } )
+		.then( () =>
```

Comment:
I don't think the theme setup is working here. If I run Headstart on a new site using Zoologist, I get different content than what I get when you sign up with this patch.